File: Test/Java/NthFiboRecursion.java
import java.lang.*;           //importing all the classes from lang package
import java.util.Scanner;

class NthFiboRecursion{
    static int fibo(int n)
    {
    if (n <= 1)
       return n;
    return fibo(n-1) + fibo(n-2);
    }
      
    public static void main (String args[])
    {
        int n;
	
	    Scanner sc = new Scanner(System.in);
	
	    System.out.print(" Enter nth place of number: ");
	    n = sc.nextInt();

        System.out.print("Nth Fibonacci number is " + fibo(n) + ".");
    }
    
}    

File: Test/CPP/NthFiboRecursion.cpp
#include<iostream>
using namespace std;
 
int fibo(int n)
{
    if (n <= 1)
        return n;
    return fibo(n-1) + fibo(n-2);
}
int main()
{
    int n;

    cout<<"Enter nth place of number: ";
    cin>>n;

    cout<<"Nth Fibonacci number is "<<fibo(n)<< "."; 
    
    return 0;
}